apps/extension/src/Approvals/ConfirmSignTx.tsx

```diff
@@ -46,7 +48,6 @@ export const ConfirmSignTx: React.FC<Props> = ({ details }) => {
 
         setStatus(Status.Completed);
       } catch (e) {
-        console.info(e);
```

Comment:
lint was complaining about this console, let me know if it need to be here that I add an lint ignore comment

Reply:
we can remove it!